src/installer/corehost/cli/json_parser.h

```diff
@@ -5,6 +5,10 @@
 #ifndef __JSON_PARSER_H__
 #define __JSON_PARSER_H__
 
+#ifdef __sun
+#define RAPIDJSON_48BITPOINTER_OPTIMIZATION 0
```

Comment:
Can you please add the reason for disabling this optimization in the comments (add the text in the commit notes).